### ISSUE TITLE
Dagster-slack is missing py.typed marker

### ISSUE THREAD

**Issue by nils-borrmann-tacto** [dagster-io/dagster#23022]
### Dagster version

dagster, version 1.7.12, dagster-slack==0.23.12

### What's the issue?

When importing `dagster_slack`, mypy hits me with the `Skipping analyzing "dagster_slack": module is installed, but missing library stubs or py.typed marker  [import-untyped]` error. This is because the `py.typed` marker is not distributed with the dagster_slack package.

Looking at the module itself, the py.typed file [is present](https://github.com/dagster-io/dagster/blob/master/python_modules/libraries/dagster-slack/dagster_slack/py.typed), but it seems to be lost in the packaging process.

### What did you expect to happen?

py.typed file should be distributed with the dagster-slack package.

### Deployment type

None

### Message from the maintainers

Impacted by this issue? Give it a 👍! We factor engagement into prioritization.

**Comment by nils-borrmann-tacto** [comment]
Hi @rexledesma! 

Thanks for the quick fix! Did this change make it into the `0.23.15` release? It looks like it should've been, but the py.typed file seems to still be missing unfortunately:

```
nilsborrmann@MacBook-Pro-3 Downloads % unzip dagster_slack-0.23.15-py3-none-any.whl
Archive:  dagster_slack-0.23.15-py3-none-any.whl
  inflating: dagster_slack/__init__.py
  inflating: dagster_slack/hooks.py
  inflating: dagster_slack/resources.py
  inflating: dagster_slack/sensors.py
  inflating: dagster_slack/version.py
  inflating: dagster_slack-0.23.15.dist-info/LICENSE
  inflating: dagster_slack-0.23.15.dist-info/METADATA
  inflating: dagster_slack-0.23.15.dist-info/WHEEL
  inflating: dagster_slack-0.23.15.dist-info/top_level.txt
  inflating: dagster_slack-0.23.15.dist-info/RECORD
```

**Comment by rexledesma** [comment]
@nils-borrmann-tacto It did not make it to our 0.23.15 release. It will be released in `0.24.0`.
